Include/internal/pycore_fileutils.h

```diff
@@ -253,6 +253,7 @@ extern int _Py_add_relfile(wchar_t *dirname,
                            size_t bufsize);
 extern size_t _Py_find_basename(const wchar_t *filename);
 PyAPI_FUNC(wchar_t *) _Py_normpath(wchar_t *path, Py_ssize_t size);
+extern wchar_t *_Py_normpathAndSize(wchar_t *path, Py_ssize_t size, Py_ssize_t *length);
```

Comment:
```suggestion
extern wchar_t *_Py_normpath_and_size(wchar_t *path, Py_ssize_t size, Py_ssize_t *length);
```

We're in `snake_case` for this name, even though we typically use PascalCase for native API.

Reply:
Thanks! I've updated the name to be snake case.